includes/lib/transifex-live-integration-rewrite.php

```diff
@@ -40,6 +40,7 @@ public function __construct( $settings, $rewrite_options ) {
 		Plugin_Debug::logTrace();
 		include_once TRANSIFEX_LIVE_INTEGRATION_DIRECTORY_BASE . '/includes/common/transifex-live-integration-validators.php';
 		include_once TRANSIFEX_LIVE_INTEGRATION_DIRECTORY_BASE . '/includes/override/transifex-live-integration-generate-rewrite-rules.php';
+		include_once __DIR__ .'/transifex-live-integration-wp-services.php';
```

Comment:
Why not `TRANSIFEX_LIVE_INTEGRATION_DIRECTORY_BASE ` as above?

Reply:
It's a mystery to me how when this class is included in tests it resolves TRANSIFEX_LIVE_INTEGRATION_DIRECTORY_BASE constant to its correct value :) . That's why I  referenced the added include relatively via path. I agree with you that for uniformity reasons though it should use TRANSIFEX_LIVE_INTEGRATION_DIRECTORY_BASE as above.